pkg/cluster/parse.go

```diff
@@ -0,0 +1,210 @@
+package cluster
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/yaml"
+
+	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+)
+
+type parsed struct {
+	objects        objects
+	Cluster        *anywherev1.Cluster
```

Comment:
why does `Cluster` field need to be exported?

Reply:
It doesn't, changed